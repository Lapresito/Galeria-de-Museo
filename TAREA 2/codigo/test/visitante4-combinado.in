crearVisitante 31 Filemon Gonzalez 29
imprimirVisitante
imprimirEdadVisitante
imprimirIdVisitante
copiarVisitante
imprimirVisitante
liberarVisitante
Fin
