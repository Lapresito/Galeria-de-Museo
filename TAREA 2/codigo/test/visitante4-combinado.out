Bienvenido al programa principal de ArteProg. Por favor ingrese su comando:
1> Visitante creadox de forma exitosa.
2> Visitante Filemon Gonzalez
Id: 31
Edad: 29
3> La edad del visitante es: 29
4> La id del visitante es: 31
5> Visitante copiado. Datos de la copia:
Visitante Filemon Gonzalez
Id: 31
Edad: 29
6> Visitante Filemon Gonzalez
Id: 31
Edad: 29
7> Visitante liberado con exito.
8> Fin.
